Scoop/NetworkManager.swift

```diff
@@ -100,13 +100,47 @@ class NetworkManager {
                 }
             }
         }
+
+    func deleteUser(
+        netid: String,
+        first_name: String,
+        last_name: String,
+        grade: String,
+        phone_number: String,
+        pronouns: String,
+        prof_pic: String,
+        prompts: [UserAnswer],
+        completion: @escaping (Result<BaseUser, Error>) -> Void
+    ) {
+        let parameters: [String: Any] = [
```

Comment:
I believe Alamofire has a custom Parameters class that we can use for the typing of this